app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -37,6 +37,8 @@
 import java.util.Arrays;
 import java.util.Calendar;
 import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
 
```

Comment:
we have uniq(list-string).sorted available, so sinple list append with duplicates would be totally fine

Reply:
Good point. I have cleaned up that code and used it